src/main/java/io/snyk/jenkins/tools/internal/DownloadService.java

```diff
@@ -5,20 +5,30 @@
 import javax.annotation.Nonnull;
 import java.io.IOException;
 import java.net.URL;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
 
 import static java.lang.String.format;
 
 public class DownloadService {
+  private static final String SNYK_DOWNLOAD_PRIMARY = "https://downloads.snyk.io/%s/%s/%s";
+  private static final String SNYK_DOWNLOAD_SECONDARY = "https://static.snyk.io/%s/%s/%s";
+  public static final String SNYK_INTEGRATION_NAME = "JENKINS_PLUGIN";
```

Comment:
Issue: This is not the correct Integration name. Please try to re-use existing constants instead of creating a new one, see PluginMetaData.java for the correct value.